AUTHORS.md

```diff
@@ -0,0 +1,49 @@
+# Credits
+
+---
+© 2024 **xDEM developers**.
+
+**xDEM** is licensed under permissive Apache 2 license (See LICENSE file).
+
+All contributors listed in this document are part of the **xDEM developers**, and their
+contributions are subject to the project's copyright under the terms of the
+[Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0).
+
+See CONTRIBUTING.md for more details on Contributor License Agreement.
```

Comment:
Given this statement, do we need to have a "Contributor License Agreement" section in `CONTRIBUTING.md`, or is this just a generic terminology?
(We can also add "the" before "Contributor License Agreement"!)

Reply:
Good review @rhugonnet : We can indeed remove this statement. We have chosen not to go to a contributor license agreement model. If needed at some point, this will be further evolution.

Reply:
@vschaffn we can remove this statement